packages/e2e-tests/playwright.config.ts

```diff
@@ -4,9 +4,21 @@ import { devices as replayDevices } from "@replayio/playwright";
 const { CI, SLOW_MO, SHARD_NUMBER } = process.env;
 
 const projects = [
+  {
+    name: "replay-chromium-custom",
+    use: {
+      launchOptions: {
+        executablePath:
+          process.env["REPLAY_BROWSER_BINARY_PATH"] ||
```

Comment:
Can we use `REPLAY_CHROMIUM_EXECUTABLE_PATH` instead, as we are using that already in other places?